openshift/templates/vc-service/api-deploy.yaml

```diff
@@ -0,0 +1,282 @@
+kind: Template
+apiVersion: template.openshift.io/v1
+metadata:
+  name: ${NAME}${SUFFIX}-deployment-template
+  annotations:
+    description: Deployment template for a FastAPI service.
+    tags: fastapi
+    iconClass: icon-python
+objects:
+
+  - kind: NetworkPolicy
+    apiVersion: networking.k8s.io/v1
+    metadata:
+      name: ${NAME}${SUFFIX}-inbound
+      labels:
+        name: ${NAME}${SUFFIX}
+        app: ${APP_NAME}${SUFFIX}
+        app-group: ${APP_GROUP}
+        role: ${ROLE}
+        env: ${TAG_NAME}
+    spec:
+      description: |
+        Allow the vcr-api and frontend to access the vc-service.
+      ingress:
+        - from:
+            - podSelector:
+                matchLabels:
+                  role: ${API_ROLE}
+                  app: ${APP_NAME}${SUFFIX}
+                  env: ${TAG_NAME}
+              namespaceSelector:
+                matchLabels:
+                  name: ${NAMESPACE}
+                  environment: ${TAG_NAME}
```

Comment:
@esune, Does the existing API need to access the new VC service?  My understanding is the new service is standalone.

Reply:
The service is standalone, but they talk to each other: the vc service posts to the webhook endpoint in the existing APi to trigger credential processing and the OrgBook api relies on the vc service to fetch the W3C credentials